app/services/edition_publisher.rb

```diff
@@ -53,4 +54,10 @@ def scheduled_for_publication?
     # Just using edition.scheduled? misses submitted editions
     edition.scheduled_publication.present?
   end
+
+  def flag_if_political_content!
+    return if edition.previously_published
```

Comment:
Looking at the implementation of `previously_published`, this probably wants to be `edition.document.published?`, otherwise a first-edition with `first_published_at` set by the user will not get its political status set.